pep-0655.rst

```diff
@@ -388,16 +397,16 @@ module.
 Rejected Ideas
 ==============
 
-Special syntax around the *key* of a TypedDict item
----------------------------------------------------
+Special notation around the *key* of a TypedDict item
+-----------------------------------------------------
```

Comment:
I think this should remain "syntax" as it discusses rejected syntax changes (same for the below usages in the same sub-section).

A

Reply:
Integrated.